java/amazon-kinesis-producer/src/main/java/com/amazonaws/services/kinesis/producer/GlueSchemaRegistrySerializerInstance.java

```diff
@@ -11,6 +11,7 @@
 public final class GlueSchemaRegistrySerializerInstance {
 
     private volatile GlueSchemaRegistrySerializer instance = null;
+    private static final String USER_AGENT_APP_NAME = "kpl";
```

Comment:
Can we add KPL version here as well?

Reply:
Is there a good way to get it?

Reply:
It's hard-coded in the C++ [core](https://github.com/awslabs/amazon-kinesis-producer/blob/master/aws/kinesis/core/kinesis_producer.cc#L41). We can just hard-code it here as well and manually update the version in our release process. I think this will be handy in the case that there's an issue with Glue your team could help let us know which version(s) are affected.